raiden/transfer/state.py

```diff
@@ -66,6 +59,7 @@
     WithdrawAmount,
     typecheck,
 )
+from raiden_contracts.tests.utils.constants import LOCKSROOT_OF_NO_LOCKS
```

Comment:
https://github.com/raiden-network/raiden-contracts/blob/acd697fec11be7e008abaf3abf638b1ceff8d253/raiden_contracts/tests/utils/constants.py#L11-L15

Why is `LOCKSROOT_OF_NO_LOCKS` called this way? while the other follow the `EMPTY_`? Are we just trying to be clever with naming than aim for consistency / simplicity?

Reply:
All `EMPTY_`'s values are `0x00000...00` but `LOCKSROOT_OF_NO_LOCKS` looks different.

Reply:
I didn't want to call `b"\xc5\xd2F\x01\x86\xf7#<\x92~}\xb2\xdc\xc7\x03\xc0\xe5\x00\xb6S\xca\x82';{\xfa\xd8\x04]\x85\xa4p"` `EMPTY_LOCKSROOT`.